cfme/fixtures/pytest_selenium.py

```diff
@@ -1137,7 +1150,11 @@ def _login_func():
                 'service evmserverd status').output)
             raise
 
-    if recycle:
+    if restart_evmserverd:
+        store.current_appliance.ssh_client.run_command("service evmserverd restart")
```

Comment:
Can we utilize this instead? https://github.com/RedHatQE/cfme_tests/blob/master/utils/appliance.py#L1238 Should we provide a wait keyword to this function so that you only need to call one line?